va/validation-authority.go

```diff
@@ -59,6 +62,43 @@ type verificationRequestEvent struct {
 	Error        string         `json:",omitempty"`
 }
 
+// TODO Update jws.go to accept jose.JsonWebKey in newVerifier
```

Comment:
Per [wiki/Boulder-Development](https://github.com/letsencrypt/boulder/wiki/Boulder-Development) this should be in the form of either `TODO(rlb...)` or `TODO(Issue #...)`